packages/sync/src/createClient.ts

```diff
@@ -3,27 +3,22 @@ import { persistCache } from "apollo-cache-persist";
 import { ApolloClient, ApolloClientOptions } from "apollo-client";
 import { DataSyncConfig } from "./config/DataSyncConfig";
 import { SyncConfig } from "./config/SyncConfig";
-import { defaultLinkBuilder as buildLink} from "./links/LinksBuilder";
+import { defaultLinkBuilder as buildLink } from "./links/LinksBuilder";
 import { PersistedData, PersistentStore } from "./PersistentStore";
-import { SyncOfflineMutation } from "./offline/SyncOfflineMutation";
 
 /**
  * Factory for creating Apollo Client
  *
  * @param userConfig options object used to build client
  */
-export const createClient = async (userConfig?: DataSyncConfig) => {
-
+export const createClient = async (userConfig?: DataSyncConfig, oldClient?: ApolloClient<NormalizedCacheObject>) => {
```

Comment:
what is `oldClient` for? It is kinda strange to tell people to pass client to function that creates client.